server/src/ngcc.ts

```diff
@@ -27,7 +27,7 @@ export async function resolveAndRunNgcc(tsconfig: string, progress: Progress): P
   if (!ngcc) {
     throw new Error(`Failed to resolve ngcc from ${directory}`);
   }
-  const index = ngcc.resolvedPath.lastIndexOf('node_modules');
+  const index = ngcc.resolvedPath.indexOf('node_modules');
```

Comment:
Yes. In short, the node_modules path under Bazel is a symlinked node_modules tree that looks like
```
node_modules/.aspect_rules_js/foo@1.2.3/node_modules/foo
```
so taking the `lastIndexOf` doesn't result in the correct path

Reply:
This would be true for users using pnpm outside of bazel as well where node_modules paths are similarly,

```
node_modules/.pnpm/foo@1.2.3/node_modules/foo
```

Reply:
I don't think I have the expertise to verify that this change is correct. I think it's important to note here that we're resolving ngcc in the developer project's `node_modules`, not ours. Does that change anything? 

To be honest, I don't know if the `lastIndexOf` was intentionally used over `indexOf` in the first place.

Reply:
Here's the commit which added this: https://github.com/angular/vscode-ng-language-service/pull/1042

Reply:
Moved this change to a pre-factor PR https://github.com/angular/vscode-ng-language-service/pull/1742